python/pyspark/pandas/namespace.py

```diff
@@ -2747,13 +2747,20 @@ def merge(
 
 
 @no_type_check
-def to_numeric(arg):
+def to_numeric(arg, errors="raise"):
```

Comment:
can we change the default to `coerce`?

Reply:
Is it okay although pandas use `raise` as default ??

I worry about that maybe the existing pandas users want to `to_numeric` just raise an Exception rather than change the value to `NaN` by default ??